src/dotnet/Core/Collections/ArrayExt.cs

```diff
@@ -92,4 +92,13 @@ public static void Deconstruct<T>(this T[] array, out T? first, out T? second, o
 
     public static void Deconstruct<T>(this T[] array, out T? first, out T? second, out T? third, out T? fourth, out T? fifth, out T[] rest)
         => (first, second, third, fourth, (fifth, rest)) = array;
+
+    public static ApiArray<T> ToApiArray<T>(this T[] array)
```

Comment:
don't we already have extension?

Reply:
![image](https://github.com/Actual-Chat/actual-chat/assets/4425748/8538ea09-4c00-4d72-b55a-43353750a3c5)